lib/unobtrusive_flash/controller_mixin.rb

```diff
@@ -50,15 +50,16 @@ def sanitize_flash(flash, displayable_flash_keys)
       end
 
       def append_flash_to_cookie(existing_cookie, flash, unobtrusive_flash_keys)
-        cookie_flash = (existing_cookie && parse_cookie(existing_cookie)) || []
-        cookie_flash += sanitize_flash(flash, unobtrusive_flash_keys)
+        cookie_flash = (existing_cookie && parse_cookie(existing_cookie))
+        cookie_flash = [] unless cookie_flash.is_a? Array
+        cookie_flash += self.sanitize_flash(flash, unobtrusive_flash_keys)
```

Comment:
Doesn't this violate [Style/RedundantSelf](http://www.rubydoc.info/gems/rubocop/0.16.0/Rubocop/Cop/Style/RedundantSelf)?